build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1029,6 +1217,50 @@ message ActionResult {
 
   // The details of the execution that originally produced this result.
   ExecutedActionMetadata execution_metadata = 9;
+
+  // The inlined contents of a
+  // [ConsoleOutputs][build.bazel.remote.execution.v2.ConsoleOutputs] message
+  // containing a series of stdout and stderr outputs in order of write from
+  // the action. The server SHOULD NOT inline `console_outputs` unless
+  // requested by the client in the
+  // [GetActionResultRequest][build.bazel.remote.execution.v2.GetActionResultRequest]
+  // message. The server MAY omit inlining, even if requested, and MUST do so if inlining
+  // would cause the response to exceed message size limits.
+  bytes console_outputs = 13;
+
+  // The digest for a blob containing the
+  // [ConsoleOutputs][build.bazel.remote.execution.v2.ConsoleOutputs] for the action,
+  // which can be retrieved from the
+  // [ContentAddressableStorage][build.bazel.remote.execution.v2.ContentAddressableStorage].
+  Digest console_outputs_digest = 14;
+}
+
+// A sequence of console outputs in the order received for a remoted process.
+// This may contain the interleaved outputs from a process in the order
+// received during execution. If the service does not implement
+// interleaving, this may contain a summary of the stdout and stderr output
+// streams as single large writes in a server-defined order.
+message ConsoleOutputs
```

Comment:
Would it make sense to decompose this PR into two parts, and first focus on getting this added? It looks like this feature is somewhat independent of the dynamic remote execution.

(Nit: Place the { on the same line as 'message ConsoleOutputs'.)

Reply:
I agree.

Reply:
Broke out into a new PR: https://github.com/bazelbuild/remote-apis/pull/173